evidence/pool.go

```diff
@@ -191,19 +343,49 @@ func (evpool *Pool) MarkEvidenceAsCommitted(height int64, evidence []types.Evide
 	}
 }
 
-// Has checks whether the evidence exists either pending or already committed
-func (evpool *Pool) Has(evidence types.Evidence) bool {
-	return evpool.IsPending(evidence) || evpool.IsCommitted(evidence)
-}
+// fastCheck leverages the fact that the evidence pool may have already verified the evidence to see if it can
+// quickly conclude that the evidence is already valid.
+func (evpool *Pool) fastCheck(ev types.Evidence) bool {
+	key := keyPending(ev)
+	if lcae, ok := ev.(*types.LightClientAttackEvidence); ok {
+		evBytes, err := evpool.evidenceStore.Get(key)
+		if err != nil {
+			evpool.logger.Error("Fastcheck Error", "err", err)
```

Comment:
the error msg is not helpful at all imho

Reply:
I'm hoping that the error message itself is telling with what actually happened. The fast check part is to let the user know where exactly it happened so they can trace what caused it

Reply:
Do you think something like `("Failed to get evidence during fastcheck", "err", err, "ev", ev)` might be better

Reply:
yes